core/templates/dev/head/galleries/gallery.html

```diff
@@ -202,6 +107,8 @@
 
   {% include 'components/exploration_creation_button_modal.html' %}
   {% include 'components/profile_link_image_directive.html' %}
+  {% include 'components/search_bar_directive.html' %}
+  {% include 'components/create_exploration_button_directive.html' %}
```

Comment:
I propose putting this one (and the corresponding JS import) in base.js/base.html, since we'll need it in most pages.